src/AnaSample.cc

```diff
@@ -281,17 +283,20 @@ void AnaSample::InitEventMap()
     else for(auto& e : m_pmts)
     {
         std::vector<double> binvar;
-        for (auto t : m_binvar)
+	//std::cout << TAG << m_binvar << std::endl;
+	for (auto t : m_binvar)
+            //std::cout << TAG << t << std::endl;
             binvar.push_back(e.GetEventVar(t));
         const int b = m_bm.GetBinIndex(binvar);
 #ifndef NDEBUG
         if(b < 0)
         {
-            std::cout << TAG << "In InitEventMap()\n"
-                      << "No bin for current PMT." << std::endl;
-            std::cout << TAG << "PMT Var: " << std::endl;
-            for(const auto val : e.GetRecoVar())
-                std::cout << TAG << "\t" << val << std::endl;
+//acraplet -> need to see what I am doing remove the warnings
```

Comment:
This is the warning when a PMT cannot be assigned to a sample bin. Why are you commenting them?